tests/src/JIT/HardwareIntrinsics/X86/Shared/AlternatingBinOpTest.template

```diff
@@ -0,0 +1,329 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
New template. Used for non-scalar intrinsics which do one operation on the odd indexes and another operation on even indexes.